tools/debian/rules

```diff
@@ -8,6 +8,12 @@ ifneq ($(shell dpkg -s libpcp3-dev >/dev/null 2>&1 && echo yes),yes)
 	CONFIG_OPTIONS = --disable-pcp
 endif
 
+# PackageKit crashes on update information on Ubuntu 16.04, which makes
+# "Software Updates" useless (LP: #1689820)
```

Comment:
Sure it is a bug in Ubuntu, and I filed it properly downstream. But this is a dealbreaker - I didn't find a workaround, and not being able to get updates makes this page entirely useless. I am not going to ship a completely broken new package, we can enable it later if/once the bug gets fixed. Note that pkgupdates does get shipped on ubuntu 17.04, just not on 16.04.

Some background: For a long time Ubuntu has used aptdaemon as a PackageKit compatible (-ish) implementation as PackageKit's apt support was rather poor in the past. In newer versions PackageKit itself has gotten much more attention, and Ubuntu moves away from aptdaemon.